matrix_sdk/src/client.rs

```diff
@@ -429,6 +430,54 @@ impl Client {
         session.as_ref().cloned().map(|s| s.user_id)
     }
 
+    /// Fetches the display name of the owner of the client.
+    ///
+    /// # Example
+    /// ```no_run
+    /// # use futures::executor::block_on;
+    /// # use matrix_sdk::Client;
+    /// # use url::Url;
+    /// # let homeserver = Url::parse("http://example.com").unwrap();
+    /// # block_on(async {
+    /// let user = "example";
+    /// let client = Client::new(homeserver).unwrap();
+    /// client.login(user, "password", None, None).await.unwrap();
+    ///
+    /// if let Some(name) = client.display_name().await {
```

Comment:
This example fails to compile because the method returns `Result<Option<String>>` instead of just `Option<String>`